app/controllers/system_settings_controller.rb

```diff
@@ -1,5 +1,6 @@
 class SystemSettingsController < ApplicationController
   before_action :set_system_setting, only: [:show, :edit, :update, :destroy]
+  before_action :set_primary_organization, only: [:show, :edit, :update, :destroy]
```

Comment:
Setting this so we can display it above view_header to help w understanding the SystemSettings are per instance, and not a bunch of records